tests/by-util/test_cp.rs

```diff
@@ -3218,6 +3218,65 @@ fn test_copy_contents_fifo() {
     assert_eq!(at.read("outfile"), "foo");
 }
 
+#[cfg(target_os = "linux")]
```

Comment:
the code is also enabled for android, why not enabling the test too?
thanks

Reply:
That was an oversight , enabled for android .